portfolio/src/test/java/com/google/sps/servlets/MessageTest.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class MessageTest {
```

Comment:
I went ahead and looked at the Message class as well. I'm curious what other functionality you think that you could add and test with this class? This is treated as a struct basically, with a compareTo defined to only rely on time created.

Is there any other useful functionality we can add to this Message class involving the other fields?

Reply:
I put the messages in an ArrayList and to use Sort() I had to implement compareTo. I can't think of any other functionality.